modules/dragster/src/test/ts/atomic/datatransfer/DataTransferCloneTest.ts

```diff
@@ -0,0 +1,92 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { KAssert } from '@ephox/katamari-assertions';
+import { PlatformDetection } from '@ephox/sand';
+import { assert } from 'chai';
+
+import { cloneDataTransfer, createDataTransfer, getDragImage } from 'ephox/dragster/datatransfer/DataTransfer';
+import { Event, getEvent, setDragstartEvent } from 'ephox/dragster/datatransfer/Event';
+import { getMode, Mode, setMode, setReadOnlyMode, setReadWriteMode } from 'ephox/dragster/datatransfer/Mode';
+
+describe('atomic.dragster.datatransfer.DataTransferCloneTest', () => {
+  const dragImage = {
+    image: document.createElement('div'),
+    x: 10,
+    y: 20
+  };
+
+  const testFile1 = new window.File([ 'Lorem ipsum' ], 'test.txt', { type: 'text/plain' });
+  const testFile2 = new window.File([ '<p>Lorem ipsum</p>' ], 'test2.html', { type: 'text/html' });
+
+  const getTestDataTransfer = (mode: Mode): DataTransfer => {
+    const dataTransfer = createDataTransfer();
+
+    setReadWriteMode(dataTransfer);
+
+    // set some test data
+    setDragstartEvent(dataTransfer);
+    dataTransfer.dropEffect = 'copy';
+    dataTransfer.effectAllowed = 'copy';
+    dataTransfer.setData('text/plain', '123');
+    dataTransfer.setData('text/html', '<p>123</p>');
+    dataTransfer.setDragImage(dragImage.image, dragImage.x, dragImage.y);
+    dataTransfer.items.add(testFile1);
+    dataTransfer.items.add(testFile2);
+
+    // set mode
```

Comment:
Stating the obvious if the code and the comment is basically the same the comment is not really needed.